requirements.txt

```diff
@@ -19,33 +29,55 @@ frozenlist==1.4.0
 idna==3.7
 importlib-metadata==6.8.0
 importlib-resources==6.1.1
+installer==0.7.0
 isoduration==20.11.0
 itsdangerous==2.1.2
+jaraco.classes==3.4.0
+jeepney==0.8.0
 Jinja2==3.1.4
 jsonpointer==2.3
 jsonschema==4.17.3
+keyring==24.3.1
 MarkupSafe==2.1.3
 mccabe==0.7.0
+more-itertools==10.2.0
+msgpack==1.0.8
 multidict==6.0.4
+packaging==24.0
+pexpect==4.9.0
+pkginfo==1.11.0
 pkgutil_resolve_name==1.3.10
+platformdirs==4.2.2
+poetry==1.8.2
```

Comment:
curious where a lot of these dependencies came from - beacon itself doesn't use poetry for dependencies (yet)

Reply:
slightly improved now, some of these changes were for debugpy, which doesn't need to be in requirements. 

However, poetry is still there, and `pip show` is not giving me any hints as to where it came from.

Reply:
it might be worth actually implementing poetry for beacon – the other services use it (even though it can be a bit finicky) and it has the advantage of declarative dependencies + split prod/dev dependencies. I used to dislike it but I've really come around to it...